packages/forklift-console-plugin/src/modules/StorageMaps/actions/StorageMapActionsDropdown.tsx

```diff
@@ -23,28 +24,45 @@ const StorageMapActionsKebabDropdown_: React.FC<StorageMapActionsDropdownProps>
 }) => {
   const { t } = useForkliftTranslation();
 
-  // Hook for managing the open/close state of the dropdown
-  const [isDropdownOpen, toggle] = useToggle();
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
```

Comment:
prefer setIsOpen(open => !open);

this is a promise way to get always latest value